README.md

```diff
@@ -18,236 +18,26 @@
   https://img.shields.io/gitter/room/sonar-scala/sbt-sonar.svg?colorB=46BC99&label=Chat
 [insightio-badge]: https://img.shields.io/badge/Insight.io-Ready-brightgreen.svg
 
-An sbt plugin which provides an easy way to integrate Scala projects with
-[SonarQube](https://www.sonarqube.org) - a tool for continuous code inspection
-and quality management :white_check_mark:.
+Sbt-sonar is an sbt plugin, which provides an easy way to integrate Scala
```

Comment:
It's just me, or that upper case `S` looks weird? I would just leave it lower case.

Reply:
Ehh, I couldn't decide about it and probably changed the convention a few times 😆 I've settled on the upper case because it's at the beginning of a new sentence, but I have mixed feelings about it. Are you suggesting we stick to `sbt-sonar` and `sonar-scala` everywhere regardless of whether it's at the beginning of a new sentence or not?

Reply:
Yeah, I have been having those mixed feelings too. But in the end, I decided that it was better to respect the name of the project.
As an example, look to the [microsite](https://neotypes.github.io/neotypes/) of **neotypes**, IMHO it doesn't look bad at all.

Reply:
ok cool, it's settled then - I'll make those changes